CanInterface/include/CanBusAccess.h

```diff
@@ -0,0 +1,33 @@
+#ifndef CANBUSACCESS_H
+#define CANBUSACCESS_H
+
+#include "CanLibLoader.h"
+#include "CCanAccess.h"
+#include <map>
+#include <string>
+
+#pragma once
+
+using namespace std;
```

Comment:
"using namespace" in an user header file seems wrong practice.

Reply:
Yep. This is actually code directly inherited from the CanModule-rev branch, therefore strictly speaking out of the scope of this effort.

But I agree with your comment and, hopefully, trivial. I'll fix it...

Reply:
done - one line change (deletion)